core/controllers/acl_decorators.py

```diff
@@ -2753,6 +2753,84 @@ def test_can_accept_suggestion(
     return generate_decorator_for_handler
 
 
+def can_update_suggestions(handler):
+    """Decorator to check whether user can update the suggestions that
+    they are allowed to review.
```

Comment:
Might it be clearer to call this decorator ``can_reviewer_update_suggestions`` then? 'can_update_suggestions' seems a bit too generic since there are other cases where users can update suggestions, and we specifically care about checking reviewers in this case--right?

Or, is this just about checking edit suggestion capabilities? Do we not already have this check for translators?

Reply:
The reviewer and the the one who made the suggestion are allowed to update the suggestion.